Db.cs

```diff
@@ -29,31 +29,59 @@ public Db(IConfiguration configuration)
             Items = Client.GetContainer("replicache-sample-todo", "items");
         }
 
-        public async Task<UInt64> GetMutationID(String accountID, String clientID)
+        public async Task RegisterStoredProcedures()
         {
-            var def = new QueryDefinition("SELECT * FROM c WHERE c.accountID = @accountID AND c.id = @id")
-                .WithParameter("@accountID", accountID)
-                .WithParameter("@id", GetReplicacheStateID(clientID));
-            await foreach (ClientState state in Items.GetItemQueryIterator<ClientState>(def))
+            string sharedPath = "js/shared.js";
+            string utilSource = $"\n// {sharedPath}\n{File.ReadAllText(sharedPath)}";
```

Comment:
When you run this are you running it on vercel or just locally? I'm kind of surprised that this File API works when deployed on vercel, but I guess it could.

Reply:
I haven't tried deploying this. `/` paths works everywhere but I guess we might need to use cwd here?